cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml

```diff
@@ -125,48 +127,45 @@ phases:
 
               # Kernel
               KERNEL_VERSION="$(uname -r)"
-              add_tag "parallelcluster:kernel" "${KERNEL_VERSION}"
+              add_tag "parallelcluster:kernel_version" "${KERNEL_VERSION}"
               append_description "kernel-${KERNEL_VERSION}"
 
               # sudo
-              add_tag "parallelcluster:sudo" "$(get_package_version "sudo")"
+              add_tag "parallelcluster:sudo_version" "$(get_package_version "sudo")"
 
               # Lustre
               LUSTRE_VERSION="$(get_package_version "lustre-client")"
-              add_tag "parallelcluster:lustre" "${LUSTRE_VERSION}"
-              append_description "${LUSTRE_VERSION}"
+              add_tag "parallelcluster:lustre_version" "${LUSTRE_VERSION}"
+              append_description "${lustre-LUSTRE_VERSION}"
```

Comment:
should it also be lustre-${LUSTRE_VERSION}?

Reply:
fixed as well